ffdc/collect_logs.sh

```diff
@@ -0,0 +1,219 @@
+#!/bin/bash
+# IBM_SHIP_PROLOG_BEGIN_TAG
+# *****************************************************************
+#
+# Licensed Materials - Property of IBM
+#
+# (C) Copyright IBM Corp. 2017,2020. All Rights Reserved.
+#
+# US Government Users Restricted Rights - Use, duplication or
+# disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
+#
+# *****************************************************************
+# IBM_SHIP_PROLOG_END_TAG
+
+BASEDIR=$(cd $(dirname "${BASH_SOURCE[0]}")/.. && pwd)
+BINDIR=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
+source $BINDIR/logging.sh
+source ${BINDIR}/../prod_env.sh
+
+DIR=vision.logs."$(date|awk '{print $4"_"$2"_"$3"_"$6}'|sed 's/:/_/'g)"
```

Comment:
Can we not use `date` formatting to get the desired date format instead of having to use both `awk` and `sed`?